compiler/z/codegen/OMRInstructionKindEnum.hpp

```diff
@@ -103,3 +103,4 @@ IsI,               ///< is I instruction, i.e. SVC
 IsIE,              ///< is IE instruction, i.e. NIAI
```

Comment:
Could you update the copyright header in this file please?

Reply:
Fixed in [force-push](https://github.com/eclipse/omr/compare/aa4d2cf9e7c421a5ea39e9f8d701fd14e58ddbca..0b3376f0ba6f553545daf6908b91dc0a9729d50e).